### PR TITLE
Implemented new Fallback System in Tablist Module

### PR DESCRIPTION
**Pull request by iPommes**
1. Part
- Fixed new FallbackSystem in Hub Module

2. Part
Fixed new Fallback System
- Added Bungeecord event for automaticly search for a new fallback if the server was stopped using /stop or Bukkit.shutdown()
- Removed old auto search in NetworkPluginRegister.java
- Made HubCloudCommand.java faster and reliabler
- Removed uneeded empty lines
- Removed old fallback String[] system
- Optimized sendToFallback() method in SimpleCloudPlayer.java
- Opimized odd shutdown() method in SpigotBootstrap.java

Fixed Cloud
- Fixed message and starting issue in "gameserver start" Command
- Fixed GameServerRegisterPacketHandler.java, now no proxy will be shown in /server

3. Part
Fixed fallback system
- Added message to player if he was moved to a fallback server (SimpleCloudPlayer.java)
- Added a message, if a fallback server isn't corretly in the config.json
- Added a message, if no fallback server is in the config.json -> also creates a default "Lobby" fallback server
- Fixed bug, where a player was moved to a fallback server, when a template is in maintenance and the player tries to join (CollectiveProxyEvents.java)

### CODE REVIEW

File: polocloud-master/src/main/java/de/polocloud/bootstrap/template/fallback/FallbackSearchService.java [hunk 1]
```diff
@@ -25,7 +27,6 @@ public FallbackSearchService(MasterConfig masterConfig) {
     public List<IGameServer> searchForTemplate(ICloudPlayer cloudPlayer, boolean hubCommand) {
```
Comment (McMaxDE) [inline polocloud-master/src/main/java/de/polocloud/bootstrap/template/fallback/FallbackSearchService.java:27]: https://hastebin.com/axinujexix.properties

Nullpointer when joining without any MinecraftServer running.

File: polocloud-master/src/main/java/de/polocloud/bootstrap/network/handler/GameServerRegisterPacketHandler.java [hunk 2]
```diff
@@ -80,15 +80,6 @@ public void handlePacket(ChannelHandlerContext ctx, Packet obj) {
                 } catch (InterruptedException | ExecutionException e) {
                     e.printStackTrace();
                 }
-            } else {
```
Comment (McMaxDE) [inline polocloud-master/src/main/java/de/polocloud/bootstrap/network/handler/GameServerRegisterPacketHandler.java:83]: Not all Server a registered in Bungeecord.

**Review by McMaxDE — CHANGES_REQUESTED** [review]
Nullpointer when joining without any MinecraftServer running:

https://hastebin.com/axinujexix.properties

Not all MinecraftServer are registered correctly in BungeeCord

File: polocloud-master/src/main/java/de/polocloud/bootstrap/template/fallback/FallbackSearchService.java (hunk 1 above)
Comment (iPommes) [reply polocloud-master/src/main/java/de/polocloud/bootstrap/template/fallback/FallbackSearchService.java:27]: Fixed! The error occurred, when the templatename from the fallback server in the config, doesn't exitst

File: polocloud-master/src/main/java/de/polocloud/bootstrap/network/handler/GameServerRegisterPacketHandler.java (hunk 2 above)
Comment (iPommes) [reply polocloud-master/src/main/java/de/polocloud/bootstrap/network/handler/GameServerRegisterPacketHandler.java:83]: Checked everything.
- On my main cloud instance, worked
- Create fresh new cloud instance, worked
- Created new template, worked
- Started new proxy, worked

**Pull request merged**